src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -763,7 +763,7 @@ private Uri CheckProtocol(Uri uri)
 
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-macos`
Invoke-WebRequest tests.Validate Invoke-WebRequest -OutFile
```powershell
Expected '127.0.0.1:8083', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 649
649:         $jsonContent.headers.Host | Should -Be $uri.Authority
```
Invoke-RestMethod tests.Validate Invoke-RestMethod -OutFile
```powershell
Expected '127.0.0.1:8083', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 2203
2203:         $jsonContent.headers.Host | Should -Be $uri.Authority
```